daprdocs/content/en/operations/configuration/increase-read-buffer-size.md

```diff
@@ -0,0 +1,60 @@
+---
+type: docs
+title: "How-To: Handle large http header size"
+linkTitle: "HTTP header size"
+weight: 6000
+description: "Configure a larger http read buffer size"
+---
+
+Dapr has a limit for the read buffer size which is defaults to 4 KB, this param limits the maximum http header size. Using bigger than default 4KB size headers should set this to a proper value. Otherwise, you may encounter a `Too big request header` service invocation error. You can change this config by defining `dapr.io/http-read-buffer-size` annotation or `--dapr-http-read-buffer-size` flag.
```

Comment:
```suggestion
Dapr has a default limit of 4KB for the http header read buffer size.  When sending http headers that are bigger than the default 4KB, you can increase this value. Otherwise, you may encounter a `Too big request header` service invocation error. You can change the http header size by using the `dapr.io/http-read-buffer-size` annotation or `--dapr-http-read-buffer-size` flag when using the CLI.
```